build.gradle

```diff
@@ -42,11 +42,7 @@ dependencies {
         compile 'com.github.reportportal:commons-model:e608d19'
     }
 
-    compile('com.nordstrom.tools:junit-foundation:11.3.0') {
-        exclude group: 'commons-beanutils', module: 'commons-beanutils'
-    }
-    compile 'commons-beanutils:commons-beanutils:1.9.4'
-
+    compile 'com.nordstrom.tools:junit-foundation:12.0.0'
```

Comment:
Note that this release of **JUnit Foundation** includes a breaking change in the `ArtifactParams` feature. The version assigned to `agent-java-junit` should reflect this change (i.e. - increment the major version).

Reply:
Well, I believe in the future 5.0.0 release we can introduce these changes.